commands/new_function.go

```diff
@@ -112,9 +115,9 @@ func runNewFunction(cmd *cobra.Command, args []string) error {
 	}
 
 	if err := os.Mkdir(functionName, 0700); err == nil {
-		fmt.Printf("Folder: %s created.\n", functionName)
+		fmt.Printf("Folder: %s is created.\n", functionName)
 	} else {
-		return fmt.Errorf("folder: could not create %s : %s", functionName, err)
+		return fmt.Errorf("Folder: %s failed to create: %s", functionName, err)
```

Comment:
Were we following a lowercase error messages convention?

Reply:
They should start lowercase yes